### PR TITLE
regenerated arm network with today's version of Autorest and latest changes made to 2016-03-30 api version of the spec

### PR DESCRIPTION
**Pull request by amarzavery**

### CODE REVIEW

**Comment by amarzavery** [comment]
@deepakswifty , @MikhailTryakhov, @romangromov - please review this PR for latest changes in  azure-arm-network

File: lib/services/networkManagement2/lib/models/expressRouteCircuitRoutesTableSummary.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ * @member {string} [neighbor] Neighbor.
+ * 
+ * @member {number} [v] V.
+ * 
+ * @member {number} [as] AS.
+ * 
+ * @member {string} [upDown] upDown.
+ * 
+ * @member {string} [statePfxRcd] statePfxRcd .
+ *
```
Comment (amarzavery) [inline lib/services/networkManagement2/lib/models/expressRouteCircuitRoutesTableSummary.js]: @deepakswifty - I think these properties should be better described in the swagger spec. As of now the end user cannot make any sense of this.

File: lib/services/networkManagement2/lib/models/expressRouteCircuitsStatsListResult.js [hunk 2]
```diff
@@ -25,8 +27,6 @@ var util = require('util');
 function ExpressRouteCircuitsStatsListResult() {
 }
 
-util.inherits(ExpressRouteCircuitsStatsListResult, Array);
-
```
Comment (amarzavery) [inline lib/services/networkManagement2/lib/models/expressRouteCircuitsStatsListResult.js]: @deepakswifty - This change means that "x-ms-pageable-operation" extension was removed from the operation that provided expressRouteCircuitsStatsListResult as the response.

Even if the operation is not pageable, it is good to have the extension.
You can explicilty specify the nextLink to null in the extension. That will not generate the nextLink but till still provide a user friendly clientside API. 

In node.js the user will directly get the array referenced by value property a.k.a value is flattened and in C# the user will get an IEnumerable<ExpressRouteCircuitStats>.

Storage does the same thing for listing storage accounts

Comment (amarzavery) [inline lib/services/networkManagement2/lib/models/expressRouteCircuitsStatsListResult.js]: https://github.com/Azure/azure-rest-api-specs/blob/master/arm-storage/2015-06-15/swagger/storage.json#L299

**Comment by DeepakRajendranMsft** [comment]
@dihan0604  please take a look at Amar's comments

**Pull request merged**